python/prophet/diagnostics.py

```diff
@@ -109,6 +107,10 @@ def map(self, func, *iterables):
                         for args in zip(*iterables)
                      ]
                      return results
+                     
+    disable_tqdm: if True it disables the progress bar that would otherwise show up when parallel=None
```

Comment:
Existing docstring was out of order - placed disable_tqdm after parallel

Reply:
Thank you!